code/modules/food/recipes_microwave.dm

```diff
@@ -1400,6 +1400,32 @@
 	reagents = list("milk" = 15, "sodiumchloride" = 5)
 	result = /obj/item/weapon/reagent_containers/food/snacks/gigapuddi/anger
 
+//LIVING PUDDI
+//This is a terrible idea.
+
+/datum/recipe/livingpuddi
```

Comment:
Really? I had no trouble with it when I tried to make one the last time I tested it, but I'll change it regardless